### PR TITLE
adding chair and tl folder

### PR DESCRIPTION
**Pull request by parispittman**
/hold
for comments. i think this is the best place to put this. documentation geared towards a certain type of contributor. i don't think the folder will be that extensive so not sure it warrants its own in the root. 

this would also house the archive for the chair/tl need to know email

### CODE REVIEW

File: contributors/chairs-and-techleads/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+mentoring, succession planning, and staffing  
+TODO  
+
+code of conduct  
+TODO  
+
+
+Communication and feedback loops:  
+Need to Know email archive [link]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/README.md]: Need to know: https://docs.google.com/document/d/1ivmV-ouim7YcTnmv21m0pP6prmj-FFZxcRBuWbT706c/edit

File: contributors/chairs-and-techleads/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+mentoring, succession planning, and staffing  
+TODO  
+
+code of conduct  
+TODO  
+
+
+Communication and feedback loops:  
+Need to Know email archive [link]  
+Monthly meeting [link]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/README.md]: meeting link: https://docs.google.com/document/d/1Jio9rEtYxlBbntF8mRGmj6Q1JAdzZ9fTDo3ru1HK_LI/edit

File: contributors/chairs-and-techleads/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Slack: #chairs-and-techleads
+Mailing list: kubernetes-sig-leads@googlegroups.com
+
+values [link]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/README.md]: values: https://github.com/kubernetes/community/blob/master/values.md

File: contributors/chairs-and-techleads/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+Communication and feedback loops:  
+Need to Know email archive [link]  
+Monthly meeting [link]
+Slack: #chairs-and-techleads
+Mailing list: kubernetes-sig-leads@googlegroups.com
+
+values [link]  
+principles [link]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/README.md]: principals: https://github.com/kubernetes/community/blob/master/governance.md#principles

File: contributors/chairs-and-techleads/meeting-host.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+this is the future home for hosting instructions: monthly sig chair/tl meeting  
+
+1. make sure both timezones are booked on the calendar. contact contribex chairs
+if there is a problem.
+2. add the date and host name to the meeting agenda [link]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/meeting-host.md]: agenda: https://docs.google.com/document/d/1Jio9rEtYxlBbntF8mRGmj6Q1JAdzZ9fTDo3ru1HK_LI/edit

**Comment by parispittman** [comment]
PTAL @mrbobbytables - addressed all of your comments and made some more changes and clarified TODOs for others to help out with the doc

**Comment by mrbobbytables** [comment]
Thanks!
/lgtm

**Pull request merged**